src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -980,6 +980,108 @@ namespace Microsoft.FSharp.Core
     /// <category>Basic Types with Units of Measure</category>
     type int64<[<Measure>] 'Measure> = int64
 
+    [<MeasureAnnotatedAbbreviation>]
+    /// <summary>The type of machine-sized signed integer numbers, annotated with a unit of measure. 
+    /// The unit of measure is erased in compiled code and when values of this type
+    /// are analyzed using reflection. The type is representationally equivalent to 
+    /// <see cref="T:System.IntPtr"/>.</summary>
+    ///
+    /// <category>Basic Types with Units of Measure</category>
+    type nativeint<[<Measure>] 'Measure> = nativeint
```

Comment:
These will need `[<Experimental("Experimental library feature, requires '--langversion:preview'")>]` for each additional API

Reply:
Done!